### PR TITLE
Use imported name of `get` and `getProperties` in `no-get` rule

### PR DESCRIPTION
**Pull request by bmish**
CC: @mongoose700

### CODE REVIEW

File: lib/rules/no-get.js [hunk 1]
```diff
@@ -123,6 +127,15 @@ module.exports = {
     }
 
     return {
+      ImportDeclaration(node) {
+        if (node.source.value === '@ember/object') {
+          importedGetName = importedGetName || getImportIdentifier(node, '@ember/object', 'get');
```
Comment (mongoose700) [inline lib/rules/no-get.js:132]: Technically someone could import `get` multiple times with different names, right? Though other lint rules should prevent that, so I'm not concerned about it.

File: lib/rules/no-get.js [hunk 2]
```diff
@@ -193,7 +206,7 @@ module.exports = {
 
         if (
           types.isIdentifier(node.callee) &&
-          node.callee.name === 'get' &&
+          node.callee.name === importedGetName &&
```
Comment (mongoose700) [inline lib/rules/no-get.js:209]: It's possible that the imported get name is shadowed by something else in a more local scope. Is there a good way to detect that?

File: lib/rules/no-get.js (hunk 1 above)
Comment (bmish) [reply lib/rules/no-get.js:132]: Yeah, this rule should cover us: https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-duplicates.md

Comment (mongoose700) [reply lib/rules/no-get.js:132]: Theoretically you could turn on this rule and not that one. I don't know why someone would though :p

File: lib/rules/no-get.js (hunk 2 above)
Comment (bmish) [reply lib/rules/no-get.js:209]: That's a good edge case. Not sure of a convenient way to detect that at the moment.

Comment (bmish) [reply lib/rules/no-get.js:209]: This rule detects that by the way: https://eslint.org/docs/rules/no-shadow

**Pull request merged**